# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../patch/no_ftello_fseeko.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./common.h.orig	Mon Aug 30 02:20:08 1999
+++ ./common.h	Wed Dec 29 09:54:31 1999
@@ -275,7 +275,7 @@
 #ifndef STDERR_FILENO
 #define STDERR_FILENO 2
 #endif
-#if HAVE_FSEEKO
+#if HAVE_FSEEKO && 0  /* This caused problems on big endian cpus */
   typedef off_t file_offset;
 # define file_seek fseeko
 # define file_tell ftello
